detect_missing_migrations.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e -o pipefail
+
+if ./manage.py migrate --fake | grep "Your models have changes that are not yet reflected" > /dev/null
+then
+    echo "Error: one or more migrations are missing:"
+    echo
+    ./manage.py migrate --fake
```

Comment:
I do not want to make this a big deal, but it would be nice to run this command only once before, save the output to a variable and than only print here.
I also understand that the case when there are migration to create is the minority, so if you think it's not worth the time, feel free to merge as is.